src/roslibpy/comm/comm_autobahn.py

```diff
@@ -3,49 +3,53 @@
 import logging
 import threading
 
-from autobahn.twisted.websocket import WebSocketClientFactory
-from autobahn.twisted.websocket import WebSocketClientProtocol
-from autobahn.twisted.websocket import connectWS
+from autobahn.twisted.websocket import (
```

Comment:
Is this `black`s doing? maybe we need to run `isort` after?

Reply:
yes, I changed also `isort` settings to be more aligned to `black` and to our style of imports. I didn't find a way to tweak both so that they produce the import style that we currently use in other projects thou and that is a bit annoying. With the current settings, at least `black` and `isort` will not step into each other's feet.